velox/common/base/PrefixSortConfig.h

```diff
@@ -23,7 +23,8 @@ namespace facebook::velox::common {
 /// Specifies the config for prefix-sort.
 struct PrefixSortConfig {
   explicit PrefixSortConfig(
-      int64_t maxNormalizedKeySize,
+      // Same with QueryConfig kPrefixSortNormalizedKeyMaxBytes.
```

Comment:
The meaning is same, but the name is different, a confusion.
It uses this config to construct `PrefixSortConfig` in https://github.com/facebookincubator/velox/blob/main/velox/exec/Driver.h#L303

Reply:
Shall we enforce to set those two explicitly?

Reply:
No, I think, I set it because we could not construct SpillConfig directly without it.
I would prefer to remove the explicit constructor and set them like
```
struct PrefixSortConfig {
  /// Max number of bytes can store normalized keys in prefix-sort buffer per
  /// entry.
  int64_t maxNormalizedKeySize = 128;

  /// PrefixSort will have performance regression when the dateset is too small.
  int32_t threshold = 130;
};
```